client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Profile Details page
![Screenshot of the Profile Details page showing 14px font size at the bottom](https://github.com/MESH-Research/Pilcrow/assets/2668987/59f829d3-62ec-4fe1-a73d-d012e06f1278)

Reply:
This now appears 16px.